scipy/optimize/_zeros_py.py

```diff
@@ -1393,3 +1394,222 @@ def toms748(f, a, b, args=(), k=1,
                           maxiter=maxiter, disp=disp)
     x, function_calls, iterations, flag = result
     return _results_select(full_output, (x, function_calls, iterations, flag))
+
+
+def chandrupatla(f, x0, x1, args=(), xtol=_xtol,
```

Comment:
Let's make the remaining arguments keyword-only.
```suggestion
def chandrupatla(f, x0, x1, *, args=(), xtol=_xtol,
```
Arguments for why we do this can be found in a lot of other PRs, especially in stats.